### ISSUE TITLE
Every pane should have a command to show it

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5775]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.609
    OS Version      : any
    R Version       : any

### Details

The View menu has commands for showing/focusing panes. Several panes don't have any menu entry and thus are difficult to reach via keyboard-only,

- Compile PDF
- Deploy
- Find
- Markers
- Presentation
- R Markdown
- Source Cpp

### See Also
This issue does not cover whether the pane gets keyboard focus after being shown this way (in general, they all should). That is tracked here: 
* let keyboard focus consistently follow switch pane keyboard shortcuts" #5652

Also closely related:
* Visually indicate which pane has focus #6225

**Comment by gtritchie** [comment]
Closely related to #5652

**Comment by R-Hannibal** [comment]
Verified Fixed on: RStudio Version - 1.4.812, R Version - 3.6.3, Windows 10 - Desktop 

All Panes now have a shortcut option in the View menu and selecting that shortcut creates the pane if it doesn't already exist (whether focus is shifted to that new pane is much more case-by-case, but that is being worked on in [5652](https://github.com/rstudio/rstudio/issues/5652)).

One discrepancy from the PR QA Notes: In addition to Show Build and Show Presentation, Show Git only appears contextually (seemingly when a version control Git Project has been opened). This behavior is the same as 1.3 and seems intended.
